### PR TITLE
docs/server update configuration.md

### PR DESCRIPTION
**Pull request by daftaupe**
Adds the missing settings found in config.js and some examples.

### CODE REVIEW

File: _docs/server/configuration.md [hunk 1]
```diff
@@ -56,17 +88,103 @@ The Lounge ships with 4 themes.
...
+[example]: /img/screenshots/example_css.png
+[example_thumb]: /img/screenshots/example_thumbnail.png (Example CSS)
+[crypto]: /img/screenshots/crypto_css.png
+[crypto_thumb]: /img/screenshots/crypto_thumbnail.png (Crypto CSS)
+[morning]: /img/screenshots/morning_css.png
+[morning_thumb]: /img/screenshots/morning_thumbnail.png (Morning CSS)
+[zenburn]: /img/screenshots/zenburn_css.png
+[zenburn_thumb]: /img/screenshots/zenburn_thumbnail.png (Zenburn CSS)
+
+## autoload
```
Comment (astorije) [inline _docs/server/configuration.md]: This is going away in https://github.com/thelounge/lounge/pull/779, mind getting rid of it in this PR?

**Comment by daftaupe** [comment]
Done, I've remove the autoload section.

File: _docs/server/configuration.md [hunk 2]
```diff
@@ -12,39 +12,71 @@ This part of the documentation will focus on the configuration of the server. St
...
+```
 
 ## port
 
-The port to listen on.
+The port to listen on. Default to `9000`.
 
 You can override this setting by starting The Lounge like this:  
-`lounge start --port 80`
+`lounge start -- --port 80`
```
Comment (astorije) [inline _docs/server/configuration.md]: This is supposed to be fixed in v2.2.0 (not released yet), so mental note to double-check that after release and update doc accordingly. Ideally, we should not need this.

Comment (daftaupe) [inline _docs/server/configuration.md]: Yes I just wrote it in order to avoid people coming and opening tickets all the time about this.

**Pull request merged**